nse_openssl.cc

```diff
@@ -574,7 +684,13 @@ static const struct luaL_Reg openssllib[] = {
 LUALIB_API int luaopen_openssl(lua_State *L) {
 
   OpenSSL_add_all_algorithms();
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
I just mixed them up here. I am currently making the commit to the SVN trunk, so this change and the merge issues will be corrected in it. I'll make an announcement in the CHANGELOG right after and write the revision number here.